src/CardInfo.js

```diff
@@ -38,23 +42,30 @@ class CardInfo extends React.Component {
     }
 
     render() {
-        let {isChanging, id, cards, statuses} = this.props;        
-        let {status, title, description, children} = findCardById(id, cards);
-        let parent = getParentCardByIdChildren(id, cards);
+        const {isChanging, id, cards, statuses, onRedirect} = this.props;      
+        const {_id, status, title, description, children} = findCardById(id, cards);
+        const parent = getParentCardByIdChildren(_id, cards);
         let statusOptions = statuses && statuses.map(value => ({value, label: value}));
 
         return (
             <div>
-                <div>
-                    {isChanging
-                        ? <Fragment>
-                            {parent && <Link to={parent._id}>{parent.title}</Link>}
+                {isChanging
+                    ? (
+                        <Fragment>
+                            {parent && (
```

Comment:
I was wrong
when you use ternary please use it as following:
{isCondition ? (
    <div></div>
) : (
    <span></span>
})
now you see what's the difference?

Reply:
ok, i will fix